webgui/scripts/codecompass/view/fileManager.js

```diff
@@ -237,6 +239,30 @@ function (on, query, style, topic, declare, Memory, Observable,
       path.unshift('project');
 
       this.set('path', path);
+      this._setFilePath(fileInfo.path);
+    },
+
+    _setFilePath : function (path) {
+      if (!path)
+        return;
+
+      this._filePath = path;
+
+      var paneWidth = util.getFullWidth(this.domNode);
+      var titleWidth = this.fileManagerTitle.offsetWidth;
```

Comment:
The reason why I didn't use the `util.getFullWidth()` function because it returns with `NaN`. It can't calculate the width of this element (computedStyle.width = auto).